app/controllers/users_controller.rb

```diff
@@ -126,7 +126,8 @@ def quick_new
     if ldap_result.nil?
       @message = 'Sorry, the login that you entered does not exist.
       You cannot create a user profile without a valid login.'
-      render(:quick_new) && return
+      render action: 'quick_new'
```

Comment:
This will be replaced with `respond_to :html, :js`, as should the line [below](https://github.com/YaleSTC/reservations/pull/1461/files#diff-4e05ad0d64e6100656b63ad1e78f32c5R138).